DomainContent/AvatarStore/Laundry/spawnLaundryBasket.js

```diff
@@ -0,0 +1,37 @@
+//
+//  spawnLaundryBasket.js
+//
+//  Created by Rebecca Stankus on 9/28/17.
+//  Copyright 2017 High Fidelity, Inc.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+//  This script creates a laundry basket to get rid of attachments after trying them on.
+(function () {
+    var _entityImport = Script.require('https://hifi-content.s3.amazonaws.com/rebecca/entityImport.js');
```

Comment:
These variables are constants and should be named following the coding standard guidelines